### PR TITLE
Add Glitchtip reverse proxy

### PR DESCRIPTION
**Pull request by TomNaessens**

### CODE REVIEW

File: host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent [hunk 1]
```diff
@@ -0,0 +1,56 @@
+upstream glitchtip {
+   server localhost:8000;
```
Comment (TomNaessens) [inline host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent:2]: Is this port OK? Where should I run it on? Is there any overview on used ports or guidelines for new applications?

File: host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    include snippets/ssl_options_preload.conf;
+
+   location / {
+       return 302 https://glitchtip.zeus.gent/;
+   }
+}
+
+server {
+    listen 10443 ssl http2;
+    server_name glitchtip.zeus.gent;
```
Comment (redfast00) [inline host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent:32]: should be glitchtip.pratchett.zeus.gent

**Review by redfast00 — CHANGES_REQUESTED** [review]

File: host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent (hunk 2 above)
Comment (TomNaessens) [reply host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent:32]: Should this be fixed here as well then? https://github.com/ZeusWPI/ansible-config/blob/master/host_files/pratchett/nginx/gamification.pratchett.zeus.gent#L32

Comment (redfast00) [reply host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent]: No, that's correct (although I now see that there should be an extra block with a port 80 http redirect to https for gamifcation.zeus.gent). The *.pratchett.zeus.gent are for "internal use" domains (we don't have to change the DNS config for them)

File: host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    server_name glitchtip.pratchett.zeus.gent;
+
+    if ($request_method = GET) {
+        return 301 https://$server_name$request_uri;
+    }
+
+    return 308 https://$server_name$request_uri;
+}
+
+server {
```
Comment (redfast00) [inline host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent:17]: Ik had dit blok nog niet gezien, dat mag ook weg, gezien we geen gltichtip.zeus.gent gaan doen

**Review by redfast00 — CHANGES_REQUESTED** [review]

File: host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent (hunk 3 above)
Comment (TomNaessens) [reply host_files/pratchett/nginx/glitchtip.pratchett.zeus.gent:17]: Gaan we dat niet doen?

**Pull request merged**